pkg/schema/di.go

```diff
@@ -102,20 +138,27 @@ func (ts *ThemaSchema) Name() string {
 // is made with Thema and CUE.
 type ThemaSchema struct {
 	// TODO figure out what fields should be here
-	Lineage thema.Lineage
+	Lineage        thema.Lineage
+	runtimeObjects []runtime.Object
 }
 
-// SchemeBuilder returns a runtime.SchemeBuilder that will accurately represent
+// GetRuntimeObjects returns a runtime.Object that will accurately represent
 // the authorial intent of the Thema lineage to Kubernetes.
-func (ts *ThemaSchema) SchemeBuilder() *runtime.SchemeBuilder {
-	panic("TODO")
+func (ts *ThemaSchema) GetRuntimeObjects() []runtime.Object {
+	return ts.runtimeObjects
+}
+
+// SetRuntimeObjects associates a thema schema with its kubernetes runtime objects
+func (ts *ThemaSchema) SetRuntimeObjects(objects ...runtime.Object) {
+	ts.runtimeObjects = append(ts.runtimeObjects, objects...)
 }
 
 // A ObjectSchema returns a SchemeBuilder. Produced by schema components
 // to make their schema available to things relying on k8s
 type ObjectSchema interface {
 	Name() string
-	SchemeBuilder() *runtime.SchemeBuilder
+	SetRuntimeObjects(...runtime.Object)
```

Comment:
i think the concern i have about doing it this way is that it means we can't reuse this as at least a base interface type for schema that arrive dynamically, e.g. from plugins.

That's not necessarily a blocker to this approach - more a point for consideration in the discussion about what constitutes a good architecture here